tests/devices/default_qubit/test_default_qubit_preprocessing.py

```diff
@@ -141,6 +141,32 @@ def circuit(x):
 
         assert dev.tracker.totals["execute_and_derivative_batches"] == 1
 
+    @pytest.mark.parametrize("interface", ("jax", "jax-jit"))
+    def test_not_convert_to_numpy_with_jax(self, interface):
+        """Test that we will not convert to numpy when working with jax."""
+
+        dev = qml.device("default.qubit")
+        config = qml.devices.ExecutionConfig(
+            gradient_method=qml.gradients.param_shift, interface=interface
+        )
+        processed = dev.setup_execution_config(config)
+        assert not processed.convert_to_numpy
+
+    def test_convert_to_numpy_with_adjoint(self):
+        """Test that we will convert to numpy with adjoint."""
+        config = qml.devices.ExecutionConfig(gradient_method="adjoint", interface="jax-jit")
+        dev = qml.device("default.qubit")
+        processed = dev.setup_execution_config(config)
+        assert processed.convert_to_numpy
+
+    @pytest.mark.parametrize("interface", ("autograd", "torch", "tf"))
+    def test_convert_to_numpy_non_jax(self, interface):
+        """Test that other interfaces are still converted to numpy."""
+        config = qml.devices.ExecutionConfig(gradient_method="adjoint", interface=interface)
+        dev = qml.device("default.qubit")
+        processed = dev.setup_execution_config(config)
+        assert processed.convert_to_numpy
```

Comment:
Can you include "jax" as an interface in the testing?

Also, I'm curious if converting to numpy with adjoint has negative effects on performance.

Reply:
I do think we could make adjoint jittable and get some nice speed boosts, but i think that would need to be a follow on task.

Reply:
Also, the jax interface is tested in `test_not_convert_to_numpy_with_jax`.